docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+## Backup Existing Network
```

Comment:
maybe
--> Backup Existing Quickstart Network

? also maybe it'd be better to just leave the whole "quickstartiness" out of the equation? This also covers or is applicable to the "host it anywhere" and "local no docker" quickstarts. I feel like that should be pointed out. Also it should cover how to upgrade the docker/docker compose-based networks